Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-gem 'activesupport', '~> 4.0'
+gem 'activesupport', '~> 5.0'
```

Comment:
This needs to remain activesupport 4.x or we'll break specs for versions that don't support it (anything < 2.2.2).